concepts/05 UI Components/SelectBox/How to Choose a Drop-Down Editor.md

```diff
@@ -0,0 +1,91 @@
+        <td><a href="/Documentation/Guide/UI_Components/TagBox/Keyboard_Support/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/Lookup/Keyboard_Support/">Yes</a></td>
+        <td>Yes</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Search box in a drop-down field</td>
+        <td>-</td>
+        <td>-</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#searchEnabled">Yes</a></td>
+        <td>-</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Display groups</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#grouped">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#grouped">Yes</a></td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Search based on user input</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxSelectBox/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxLookup/Configuration/#minSearchLength">Yes</a></td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxAutocomplete/Configuration/#minSearchLength">Yes</a></td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Multiple selection</td>
+        <td>-</td>
+        <td><a href="/Documentation/ApiReference/UI_Components/dxTagBox/Configuration/#showSelectionControls">Yes</a></td>
+        <td>-</td>
+        <td>-</td>
+        <td>Requires custom configuration</td>
+    </tr>
+    <tr>
+        <td>Paging</td>
+        <td><a href="/Documentation/Guide/UI_Components/SelectBox/Enable_Paging/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/TagBox/Enable_Paging/">Yes</a></td>
+        <td><a href="/Documentation/Guide/UI_Components/Lookup/Enable_Paging/">Yes</a></td>
+        <td>-</td>
+        <td>Requires custom configuration</a></td>
+    </tr>
+</table>
+
+The DropDownBox UI component's API allows you to display custom content in its drop-down field. This API requires that you manually implement other components within DropDownBox and their features (see [dropDownOptions](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#dropDownOptions). 
+
+Refer to the [guides](/Documentation/Guide/UI_Components/DropDownBox/Overview/) and [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) sections for the DropDownBox to see more available customizations. 
```

Comment:
> The DropDownBox UI component's API allows you to display custom content in its drop-down field. This API requires that you manually implement other components within DropDownBox and their features (see [dropDownOptions](/Documentation/ApiReference/UI_Components/dxDropDownBox/Configuration/#dropDownOptions). 

These are just facts. They are not good, nor bad. But in this article, simple facts won't do. We need to compare editors with each other, give some recommendations, and substantiate them.

```suggestion
Unlike the rest of the editors, DropDownBox allows you to put any content in its drop-down field. This feature makes it more versatile but at the same time requires more effort from the developer. We recommend using DropDownBox in those advanced use cases that cannot be satisfied by other editors. Refer to the DropDownBox [help topics](/Documentation/Guide/UI_Components/DropDownBox/Overview/) and [demos](https://js.devexpress.com/Demos/WidgetsGallery/Demo/DropDownBox/SingleSelection/) for further information.
```